README.md

```diff
@@ -1,16 +1,305 @@
+		return Math.sqrt(a * a + b * b);
+	}
+}
+```
+
+To access, these static methods:
+
+```js
+let num = CommonMath.triple(3);
+num; // => 27
+let c = CommonMath.findHypotenuse(3, 4);
+c; // => 5
+```
+
+This sort of `class` might be useful in many different situations, but we don't
+ever need an _instance_ of it.
+
+## Define `get` Keyword in JavaScript Class Context
+
+Often, when writing methods for a `class`, we want to return information
+derived from that instance's properties. In the Square `class` example, `area()`
+returns a calculation based on `this.sideLength`, and `areaMessage()` returns
+a `String`.
+
+In modern JavaScript, new syntax, `get`, has been introduced to `class`es
+to distinguish methods which serve a specific purpose to 'get' data from an
+instance.
+
+The `get` keyword turns a method into a 'pseudo-property', that is - it allows
+us to write a method that interacts like a property. To use `get`, write a
+`class` method like normal, preceded by `get`:
+
+```js
+class Square {
+	constructor(sideLength) {
+		this.sideLength = sideLength;
+	}
+
+	get area() {
+		return this.sideLength * this.sideLength;
+	}
+}
+```
+
+As a result of this, `area` will now be available as though it is a
+property just like `sideLength`:
+
+```js
+let square = new Square(5);
+square.sideLength; // => 5
+square.area; // => 25
+```
+
+If you try to use `this.area()`, you'll receive a TypeError - `area` is no
+longer considered a function!
+
+This may seem strange - you could also just write the following and achieve the
+same result:
+
+```js
+class Square {
+	constructor(sideLength) {
+		this.sideLength = sideLength;
+		this.area = sideLength * sideLength;
+	}
+}
+```
+
+This is valid code, but what we've done is load our `constructor` with more
+calculations. The main benefit to using `get` is that your `area` calculation
+isn't actually run until it is accessed. The 'cost' of calculating is offset,
+and may not be needed at all if `this.area` is never called. For calculations or
+processes that are costly, this can be a handy tool.
+
+## Define `set` Keyword in JavaScript Class Context
+
+In the previous section we saw we could write the following after employing `get`:
+
+```js
+let square = new Square(5);
+square.sideLength; // => 5
+square.area; // => 25
+```
+
+In the `Square` `class`, we only have one real property,
+`this.sideLength`. If we want to change that directly, we would write
+`this.sideLength =` whatever value we assign it.
+
+```js
+let square = new Square(5);
+square.sideLength; // => 5
+square.area; // => 25
+square.sideLength = 10;
+```
+
+Which will change the value returned by `area`:
+
+```js
+square.area; // => 100
+```
+
+At the moment, although `area` looks and behaves like a property, we can't
+assign it a new value like a real property. Using `get` to create a
+pseudo-prototype is only half the story, since it is only used for retrieving
+data from an instance.
+
+To change data, we have `set`.
```

Comment:
Possibly introduce `set` earlier in this section. There's not a very smooth segue from the `get` section into `set`.